aspnetapp/README.md

```diff
@@ -0,0 +1,27 @@
+﻿aspnet-framework:4.6.2 Sample
+====================
+
+This sample demonstrates how to dockerize an ASP.NET Web Form app that uses .NET Framework 4.6.2 by adding a `Dockerfile` to it and running the image in [Docker for Windows](https://docs.docker.com/docker-for-windows/) with a Windows Container. To complete this example you must have [Windows 10](https://www.microsoft.com/en-us/windows/get-windows-10) (or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server)), [Docker for Windows](https://docs.docker.com/docker-for-windows/), [Visual Studio](https://www.visualstudio.com/vs/), and [Git](https://git-scm.com/) installed.
+
+## Instructions
+
+> **Note:** You must use Windows Containers on [Docker for Windows](https://docs.docker.com/docker-for-windows/) to run this image. Be sure to check that you are properly switched to Windows Containers. Do this by opening the system tray up arrow and right clicking on the Docker whale icon for a popup menu. In the popup menu make sure you select 'Switch to Windows Containers'. 
```

Comment:
Should we also note that this requires a minimum Docker for Windows version 1.13, which is currently in Beta, and Windows 10 Anniversary edition with container support enabled?